common/authorization/authorizer.go

```diff
@@ -103,3 +106,28 @@ func GetAuthProviderClient(privateKey string) (clientworker.AuthorizationProvide
 type FilteredRequestBody interface {
 	SerializeForLogging() (string, error)
 }
+
+func validatePermission(claims *JWTClaims, attributes *Attributes, data map[string]string) error {
+	groups := ""
+	switch attributes.Permission {
```

Comment:
does existing unit tests cover all cases?

Reply:
this part was not covered directly, added dedicated unit tests to this function